R/mobr.R

```diff
@@ -569,7 +569,7 @@ effect_SAD_continuous = function(out, group_sad, env_levels, corr, nperm){
         cor(x, env_levels, method=corr))
     # Null test
     overall_sad_lumped = as.numeric(colSums(group_sad))
-    meta_freq = SpecDist(overall_sad_lumped)$probability
+    meta_freq = overall_sad_lumped[overall_sad_lumped > 0] / sum(overall_sad_lumped)
```

Comment:
same advice here let's go with `sp_freq` which more clearly communicates exactly what this vector is. Also let's make this change at line 618